src/Module.zig

```diff
@@ -782,18 +789,16 @@ pub const Scope = struct {
         /// function.
         pub const Inlining = struct {
             /// Shared state among the entire inline/comptime call stack.
-            shared: *Shared,
+            shared: *IShared,
             /// We use this to count from 0 so that arg instructions know
             /// which parameter index they are, without having to store
             /// a parameter index with each arg instruction.
             param_index: usize,
             casted_args: []*Inst,
             merges: Merges,
 
-            pub const Shared = struct {
+            pub const IShared = struct {
```

Comment:
What's the `I` stand for? Presumably "Inlining"? That would be redundant. The full name is changing from

`Module.Scope.Block.Inlining.Shared`

to

`Module.Scope.Block.Inlining.IShared`

Reply:
I thought the scoping would not allow it. And if it does I was being cautious for a future when it will not. See #6969